crates/sui-storage/src/object_store/util.rs

```diff
@@ -280,9 +282,43 @@ pub fn get_path(prefix: &str) -> Path {
     Path::from(prefix)
 }
 
+// Snapshot MANIFEST file is very simple. Just a newline delimited list of all paths in the snapshot directory
+// this simplicty enables easy parsing for scripts to download snapshots
+pub async fn write_snapshot_manifest(
+    dir: &Path,
+    local_disk: Arc<DynObjectStore>,
+) -> Result<(), object_store::Error> {
+    let mut file_names = vec![];
+    let mut paths = local_disk.list(Some(dir)).await?;
+    while let Some(res) = paths.next().await {
+        if let Ok(object_metadata) = res {
+            // trim the "epoch_XX/" dir prefix here
+            let path_str = object_metadata.location.to_string();
+            let mut path_parts: Vec<&str> = path_str.split('/').collect();
+            path_parts.remove(0);
+
+            file_names.push(path_parts.join("/"));
+        } else {
+            return Err(res.err().unwrap());
+        }
+    }
+
+    let bytes = Bytes::from(file_names.join("\n"));
+    put(
+        &Path::from(format!("{}/{}", dir, MANIFEST_FILENAME)),
```

Comment:
Is this `MANIFEST` file outside the epoch_XX path? If inside, then we may need to change the name because rocksdb has its own manifest with the same name

Reply:
Should be in the path:
```
assert!(input_path.join("epoch_0").join(MANIFEST_FILENAME).exists());
```